### DISCUSSION TITLE
Accidentally found that TENGAI's ROM can be used directly

### DISCUSSION THREAD

**Discussion by evoqoo** [farmerbb/RED-Project#93] [General]
\DATA\TENGAI in the game installation directory,
Just compress the file inside into a zip and name it tengaij.zip, then you can use fbneo core to play it in Ratroarch.
Not sure if there is a difference from the original ROM.

**Comment by farmerbb** [comment]
Same thing can be done with Samurai Aces - compress everything inside the SNGKACE folder and name it `sngkace.zip`.
